src/ResourceManager/ServiceBus/Commands.ServiceBus/Models/ActionAttributes.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
```

Comment:
@v-Ajnava no license header for this file. Please make sure there are no other files missing the header.